web/tests/filter.test.js

```diff
@@ -147,6 +147,7 @@ test("basics", () => {
     assert.ok(!filter.is_conversation_view());
 
     terms = [
+        {operator: "with", operand: 17},
         {operator: "channel", operand: "foo"},
         {operator: "topic", operand: "bar"},
         {operator: "search", operand: "pizza"},
```

Comment:
I don't like the fact that this effectively changes an existing test for the `channel/topic/search` operator group.

Reply:
this must've been a mistake. There's already a separate test for `with` below.